Library/Formula/ucspi-tools.rb

```diff
@@ -0,0 +1,18 @@
+class UcspiTools < Formula
+  homepage "https://github.com/younix/ucspi/blob/master/README.md"
+  url "https://github.com/younix/ucspi/archive/v1.2.tar.gz"
+  sha1 "38a708efd6d72e0d9d077efb15477763bdea39b0"
+
+  depends_on "pkg-config" => :build
+  depends_on "ucspi-tcp"
+  depends_on "libressl"
+
+  def install
+    system "make", "PREFIX=#{prefix}", "install"
+  end
+
+  test do
+    out = shell_output("#{bin}/socks 2>&1", 1)
+    assert_equal "tcpclient PROXY-HOST PROXY-PORT socks HOST PORT PROGRAM [ARGS...]\n", out
```

Comment:
It manages to resolve GitHub normally (that's how it downloaded this change), seems there may be an issue with the tool invocation?